app/com/arpnetworking/metrics/portal/scheduling/JobCoordinator.java

```diff
@@ -256,7 +225,7 @@ public Receive createReceive() {
 
                     getContext().getSystem().scheduler().scheduleOnce(
                             scala.concurrent.duration.Duration.Zero(),
-                            createAntiEntropyRunnable(),
+                            this::runAntiEntropy,
                             getContext().dispatcher());
```

Comment:
I think this could whole block even be reduced to `runAntiEntropy()`. I think there's no reason for this to be spun off into the background, given that the `_currentlyExecuting` flag ensures that only one of these anti-entropy scans can be running at a time.